pkg/executor/util/util.go

```diff
@@ -128,3 +133,39 @@ func GetSpecFromConfigMap(ctx context.Context, kubeClient kubernetes.Interface,
 
 	return &additionalSpec, err
 }
+
+func GetObjectReaperInterval(logger *zap.Logger, executorType fv1.ExecutorType, defaultReaperInterval int) int {
+
```

Comment:
Line no 141-151 and line no 155-164 seems almost duplicate code.

Probably we can refactor code here a bit.

Let's say we define function.
```
func getValueFromEnv(envVar string) (int, error)
```

Then in `func GetObjectReaperInterval(****) int`
1. We first call `getValueFromEnv(getExecutorEnvVariableName(executorType))`, if err not nil return value fetched
2. Then we can getValueFromEnv(""OBJECT_REAPER_INTERVAL"), if err not nil return value fetched.
3. Otherwise return the default value.

This would make code bit easy to read.

Reply:
What do you think if I create `func getUIntValueFromEnv(envVar string) (uint, error)`, and check that env var exists and can be parsed to uint in this function? Looks like good util method.

Or use something like this https://pkg.go.dev/go.deanishe.net/env@v0.5.1#GetUint

Reply:
Yes, this is also interesting. 

In that case, it would be a very simple and easy call,

```
return GetUint(getExecutorEnvVariableName(executorType), GetUint("OBJECT_REAPER_INTERVAL", defaultValue))
```

However, if we can just write a function of the same essence for this purpose that would be great, rather than having another import.

`GetUIntValueFromEnv` is better name considering function same as `GetUint`.

Reply:
Did it.